ppr-ui/src/store/actions/mhr-transfer-actions-model.ts

```diff
@@ -16,6 +16,10 @@ export const setMhrTransferOwnLand: ActionIF = ({ commit }, isOwnLand: boolean):
   commit('mutateMhrTransferOwnLand', isOwnLand)
 }
 
+export const setMhrTransferSubmittingPartyKey: ActionIF = ({ commit }, { key, value }): void => {
+  commit('mutateMhrTransferSubmittingPartyKey', { key, value })
+}
+
```

Comment:
Just seeing this external actions file now. 
I think if you were going to create this additional action file specific for Transfers, you should refactor all the actions specific to Transfers, not have a few here and a few there. It's pretty confusing.

Reply:
Yea, the intention was to separate actions as the original file started to grow. I tried to keep all the Transfers-related actions in this file. Let me take a look if anything could be moved here, thanks!

Reply:
A good intention for sure, i think we just need to take it all the way to clear up any questions for future development.

Reply:
Re-organized the Transfer actions.